group_vars/location_huette15/networks.yml

```diff
@@ -1,39 +1,69 @@
 ---
-ipv6_prefix: '2001:bf7:830:9a00::/56'
 
+# alt, löschen:
+# 2001:bf7:830:9a00::/56
+# 10.31.77.200/30
+# 10.31.77.196/30
+# 10.31.112.192/26
+# 10.31.112.128/27
+# 10.31.77.13
+# 10.31.77.55
+# 10.31.77.56
 
-# Mesh: 10.31.77.200/30
+ipv6_prefix: 2001:bf7:820:700::/56
+
+# mesh: 10.31.82.192/28
+# mgmt: 10.31.107.224/28
+# dhcp: 10.31.166.128/26
 networks:
+
   - vid: 20
     role: mesh
     name: mesh_core
-    prefix: 10.31.77.200/32
+    prefix: 10.31.82.193/32
     ipv6_subprefix: -1
+
+  - vid: 21
+    role: mesh
+    name: mesh_huette
+    prefix: 10.31.82.194/32
+    ipv6_subprefix: -2
     mesh_metric: 1024
-    mesh_ap: huette15-core
+    mesh_ap: huette15-ap1
     mesh_radio: 11a_standard
     mesh_iface: mesh
 
   - vid: 40
     role: dhcp
     inbound_filtering: true
     enforce_client_isolation: true
-    prefix: 10.31.112.192/26
+    prefix: 10.31.166.128/26
     ipv6_subprefix: 0
     assignments:
       huette15-core: 1
 
   - vid: 42
     role: mgmt
-    prefix: 10.31.77.196/30
+    prefix: 10.31.107.224/28
     gateway: 1
     dns: 1
     ipv6_subprefix: 1
     assignments:
       huette15-core: 1
+      huette15-ap1: 2
+
+  - vid: 50
+    role: ext
+    name: easybell
+    untagged: true
+    tunnel_wan_ip: 192.168.178.2/24
+    tunnel_wan_gw: 192.168.178.1
+    tunnel_connections: 2
+    tunnel_timeout: 600
+    tunnel_mesh_prefix_ipv4: 10.31.82.192/28
```

Comment:
2 tunnel connections will use 2 addresses of that block, but you already use the first 2 for vid 20 and vid 21. I think you need to split the net.

Reply:
It checks which addresses are already in use and picks ones which are free :-)

```
13: wg_51820: <POINTOPOINT,NOARP,UP,LOWER_UP> mtu 1412 qdisc noqueue state UNKNOWN group default qlen 1000
    link/none 
    inet 10.31.82.192/32 scope global wg_51820
       valid_lft forever preferred_lft forever
    inet6 fe80::2/64 scope link 
       valid_lft forever preferred_lft forever
14: wg_51821: <POINTOPOINT,NOARP,UP,LOWER_UP> mtu 1412 qdisc noqueue state UNKNOWN group default qlen 1000
    link/none 
    inet 10.31.82.195/32 scope global wg_51821
       valid_lft forever preferred_lft forever
    inet6 fe80::2/64 scope link 
       valid_lft forever preferred_lft forever

```

(Although it probably shouldn't pick 192 in this case, but that's another matter.